wireless/gs2200m/gs2200m_main.c

```diff
@@ -389,17 +406,17 @@ static int usrsock_request(int fd, FAR struct gs2200m_s *priv)
       return -EIO;
     }
 
-  assert(handlers[com_hdr->reqid].hdrlen < sizeof(hdrbuf));
+  assert(handlers[com_hdr->reqid].hdrlen < sizeof(req));
```

Comment:
@anchao 

I found that the assertion happend here.
I confirmed the following condition works.

```
  assert(handlers[com_hdr->reqid].hdrlen <= sizeof(req));
```

Reply:
Thanks for your quick reply, I will update here, will the spresense works properly if this change included?

Reply:
> Thanks for your quick reply, I will update here, will the spresense works properly if this change included?

Yes. (I'm still checking the ntp client stack corruption https://github.com/apache/incubator-nuttx-apps/pull/957 but the issue does not relate to this PR)